core/src/elements/ons-back-button.js

```diff
@@ -74,15 +74,17 @@ class BackButtonElement extends BaseElement {
    *  [ja]バックボタンの見た目を指定します。[/ja]
    */
 
-  createdCallback() {
-    contentReady(this, () => {
-      if (!this.hasAttribute('_compiled')) {
-        this._compile();
-      }
+  constructor(self) {
+    self = super(self);
+
+    contentReady(self, () => {
+      self._compile();
```

Comment:
In that case we are forced to always extend the same element, right? We cannot do `extend HTMLLabelElement`, for example. Not sure if this is well supported in the polyfill anyway.